src/sentry/south_migrations/0287_auto__add_field_releaseproject_new_groups.py

```diff
@@ -0,0 +1,705 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'ReleaseProject.new_groups'
+        db.add_column('sentry_release_project', 'new_groups',
+                      self.gf('sentry.db.models.fields.bounded.BoundedPositiveIntegerField')(null=True),
```

Comment:
no, i just generated this before adding the default and then was going to generate another migration for that, but it seems like south doesn't count that as a migration worthy change (i get "Nothing seems to have changed." when trying)